### PR TITLE
Release version 1.4.0

### PR DESCRIPTION
**Pull request by sauravvishal8797**
Changes: Release version V2: 1.4.0

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/MyApplication.java [hunk 1]
```diff
@@ -36,20 +39,30 @@
...
 
+        StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
+        StrictMode.setVmPolicy(builder.build());
+
+        if (LeakCanary.isInAnalyzerProcess(this)) {
+            // This process is dedicated to LeakCanary for heap analysis.
+            // You should not init your app in this process.
+            return;
+        }
+        refWatcher = LeakCanary.install(this);
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/MyApplication.java]: Leak canary installation not required here. You can check the condition with the help of boolean isPublished and then initialize it.

**Comment by sauravvishal8797** [comment]
@anantprsd5 Did the changes. Please review.

**Comment by mariobehling** [comment]
I don't understand why this is called version 2.0. What are the major changes that would justify this? Please go back to naming it 1.xx. Thanks

**Comment by sauravvishal8797** [comment]
Sure @mariobehling . Have renamed it to version 1.4.0.

**Comment by anantprsd5** [comment]
@sauravvishal8797 The PR seems fine. Great work. Are there any issues which are yet to be solved before the new release or is it ready to get merged?

**Comment by sauravvishal8797** [comment]
@anantprsd5 I have updated the PR with the required fixes, please review. Now I think it is ready to be merged.

**Comment by anantprsd5** [comment]
The app crashes while editing photos. Please see to it.
```
FATAL EXCEPTION: main
                                                                      Process: org.fossasia.phimpme, PID: 23108
                                                                      java.lang.NullPointerException: Attempt to invoke virtual method 'void com.squareup.leakcanary.RefWatcher.watch(java.lang.Object)' on a null object reference
                                                                          at org.fossasia.phimpme.editor.fragment.BaseEditFragment.onDestroy(BaseEditFragment.java:28)
                                                                          at org.fossasia.phimpme.editor.fragment.MainMenuFragment.onDestroy(MainMenuFragment.java:57)
                                                                          at android.support.v4.app.Fragment.performDestroy(Fragment.java:2434)
                                                                          at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1442)
                                                                          at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1528)
                                                                          at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:753)
                                                                          at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2363)
                                                                          at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2149)
                                                                          at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2103)
                                                                          at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2013)
                                                                          at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:710)
                                                                          at android.os.Handler.handleCallback(Handler.java:790)
                                                                          at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                          at android.os.Looper.loop(Looper.java:164)
                                                                          at android.app.ActivityThread.main(ActivityThread.java:6494)
                                                                          at java.lang.reflect.Method.invoke(Native Method)
                                                                          at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
06-09 02:30:36.411 23108-23108/org.fossasia.phimpme E/UncaughtException: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.squareup.leakcanary.RefWatcher.watch(java.lang.Object)' on a null object reference
                                                                             at org.fossasia.phimpme.editor.fragment.BaseEditFragment.onDestroy(BaseEditFragment.java:28)
                                                                             at org.fossasia.phimpme.editor.fragment.MainMenuFragment.onDestroy(MainMenuFragment.java:57)
                                                                             at android.support.v4.app.Fragment.performDestroy(Fragment.java:2434)
                                                                             at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1442)
                                                                             at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1528)
                                                                             at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:753)
                                                                             at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2363)
                                                                             at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2149)
                                                                             at android.support.v4.app.FragmentManagerImpl.optimizeAndExecuteOps(FragmentManager.java:2103)
                                                                             at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2013)
                                                                             at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:710)
                                                                             at android.os.Handler.handleCallback(Handler.java:790)
                                                                             at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                             at android.os.Looper.loop(Looper.java:164)
                                                                             at android.app.ActivityThread.main(ActivityThread.java:6494)
                                                                             at java.lang.reflect.Method.invoke(Native Method)
                                                                             at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:440)
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

**Comment by sauravvishal8797** [comment]
@anantprasad Have resolved the crash issue and a few other issues, the app is working fine now.

**Comment by anantprsd5** [comment]
Twitter link in the about section is broken. Please fix it. Rest everything seems fine.

**Pull request merged**